parsetag.go

```diff
@@ -176,6 +175,7 @@ func (tag Tag) Fill(model interface{}, opts ...FillOptArg) error {
 					return true
 				}
 				value = elements[i]
+				delete(kv, value) // exclude from rest match
```

Comment:
@muir  This probably not good enough

for case 
``` 
struct {
    	T6 bool   `xyz:"first,first" want:"{\"Name\":\"first\",\"First\":true}"`
        T7 bool   `xyz:"first,!first" want:"{\"Name\":\"first\",\"First\":false}"`
}
```

with this line `delete(kv, value)` will make second `first` never match. But without this line, will make the second `first` always match.